### ISSUE TITLE
dont work in Call of Duty Black Ops III

### ISSUE THREAD

**Issue by jonaand** [x360ce/x360ce#274]
Hello! have windows 10 64bit, and try x360ce 64bit in
Call of Duty Black Ops III
and not work
any suggestion
thanks

**Comment by Squall-Leonhart** [comment]
don't care, pc shooters should be played with a kb/mouse.

**Comment by jonaand** [comment]
@Squall-Leonhart you are wrong
i play all my games shooter with joystick
are you crazy?

**Comment by EJocys** [comment]
I've tried to control jets and helicopters in Battlefield 4 with joystick. Saw some people to master joystick in FPS games to pro levels and successfully killing many mouse pushers. Thing is that, mouse, in combination with "muscle memory", have advantage of nearly instant targeting.

You should look for solution or create topic about it here http://ngemu.com/forums/x360ce.140

**Comment by nizamzz** [comment]
yeah okk .. I want to help, in this regard, Where to send pictures,???

**Comment by kissingsdw** [comment]
Hey did u manage to make it work?

**Comment by nizamzz** [comment]
yes.. easy .. cod.. no problem gamepad.. and fast for game,  fps fix have.. easy mod..
